### ISSUE TITLE
Is this a Flux-supported project, or is this just the repo from the site?

### ISSUE THREAD

**Issue by itsjoshuagraves** [jefferyleo/f.lux#8]
I'm wondering what kind of support to expect in the future, if any, too.

**Comment by JohnDDuncanIII** [comment]
to me, this looks like this person just mirrored the download from the site. doesn't look official at all.

**Comment by gvoysey** [comment]
i agree, this looks like a blind fork.  Which is nice!   I found https://justgetflux.com/forum/topic/1267/sha-1-for-ios-sideload  , which posted official SHAs.  It's not clear if those SHAs are for the compiled binary or the zip of the source.

**Comment by JohnDDuncanIII** [comment]
Looks like it's for the zip (which I still had in my trashcan).
![screen shot 2015-11-14 at 11 50 34 am](https://cloud.githubusercontent.com/assets/9257785/11164721/2edf0f78-8ac6-11e5-9655-cc888c2771d5.png)

**Comment by gvoysey** [comment]
see #11 : you can drill down in the link provided from the f.lux forums and get shas for individual files as well. primarily one should be concerned about the state of the precompiled blob `iflux`, which we can prove has the same hash as it did when released by its creators.

**Comment by fulldecent** [comment]
@JohnDDuncanIII  and which commit was that?

**Comment by JohnDDuncanIII** [comment]
that was the original zip file that was hosted on the f.lux website, before it was pulled

**Comment by christiangenco** [comment]
I compared the verified `f.lux-xcode-master.zip` file:

``` bash
$ openssl sha1 f.lux-xcode-master.zip
SHA1(f.lux-xcode-master.zip)= f1ee4e38eddc467e7fbfe5708841bbc84f520d7e
```

With [the current branch of this repo](https://github.com/jefferyleo/f.lux/commit/cd8c20bedc04f5bdcb0ebd898fffe668a2e06044) and got this:

```
$ diff -rq f.lux-master f.lux-xcode-master
Only in f.lux-xcode-master: AppIcon29x29@2x.png
Only in f.lux-xcode-master: AppIcon29x29@3x.png
Only in f.lux-xcode-master: AppIcon40x40@2x.png
Only in f.lux-xcode-master: AppIcon40x40@3x.png
Only in f.lux-xcode-master: AppIcon60x60@2x.png
Only in f.lux-xcode-master: AppIcon60x60@3x.png
Files f.lux-master/Info.plist and f.lux-xcode-master/Info.plist differ
Files f.lux-master/README.md and f.lux-xcode-master/README.md differ
Only in f.lux-master/flux.beta/Assets.xcassets/AppIcon.appiconset: AppIcon29x29@2x.png
Only in f.lux-master/flux.beta/Assets.xcassets/AppIcon.appiconset: AppIcon29x29@3x.png
Only in f.lux-master/flux.beta/Assets.xcassets/AppIcon.appiconset: AppIcon40x40@2x.png
Only in f.lux-master/flux.beta/Assets.xcassets/AppIcon.appiconset: AppIcon40x40@3x.png
Only in f.lux-master/flux.beta/Assets.xcassets/AppIcon.appiconset: AppIcon60x60@2x.png
Only in f.lux-master/flux.beta/Assets.xcassets/AppIcon.appiconset: AppIcon60x60@3x.png
Files f.lux-master/flux.beta/Assets.xcassets/AppIcon.appiconset/Contents.json and f.lux-xcode-master/flux.beta/Assets.xcassets/AppIcon.appiconset/Contents.json differ
Only in f.lux-xcode-master/flux.beta: README.md
Files f.lux-master/iflux.xcodeproj/project.pbxproj and f.lux-xcode-master/iflux.xcodeproj/project.pbxproj differ
```

ie: the sha1 sums are different, but the files that have been changed don't look particularly important.

The mystery `iflux` binary looks like it's identical:

``` bash
$ openssl sha1 f.lux-master/iflux
SHA1(f.lux-master/iflux)= 6d2468410a760bef01ca9cb821a40f87b1992936
$ openssl sha1 f.lux-xcode-master/iflux
SHA1(f.lux-xcode-master/iflux)= 6d2468410a760bef01ca9cb821a40f87b1992936
```

tl;dr: as of [the current commit at the time of writing this comment](https://github.com/jefferyleo/f.lux/commit/cd8c20bedc04f5bdcb0ebd898fffe668a2e06044), this repo looks like it has been significantly altered from the version that was published on the official [f.lux sideloading post](https://justgetflux.com/sideload/).
